api/helpers/gamestate/check-game-state-for-win.js

```diff
@@ -0,0 +1,71 @@
+const GameStatus = require('../../../utils/GameStatus.json');
+
+module.exports = {
+  friendlyName: 'Check Game State for win',
+
+  description: 'Determines if the game has winner and updates game status and winner if so',
+
+  inputs: {
+    game: {
+      type: 'ref',
+      description: 'Game state object',
+      required: true,
+    },
+    players: {
+      type: 'ref',
+      description: 'Array of combined game players as well as gameState players',
+      required: true,
+    },
```

Comment:
I already had to combine and make a players to send over because the client expects an array of players.. I figured it would be better to just send the necessary data since we already needed to manipulate it.

I'm game to use the game and gameState object if it will be more clear